collections/_components/modal.html

```diff
@@ -1,7 +1,511 @@
 ---
 title: Modal
 layout: layout-sidenav
+description: The modal component inform users about a specific task and may contain critical information which users then have to make a decision. E.g Accepting the T&Cs during signup
 category: general
-react: true
+react-link: https://....
+github-link: https://....
+toc: true
 ---
-{% include page-component/component-header.html %}
\ No newline at end of file
+
+{% capture modal-design-tokens %} $modal-inner-padding:$spacer !default;
+$modal-footer-margin-between:.5rem !default; $modal-dialog-margin:.5rem
+!default; $modal-dialog-margin-y-sm-up:1.75rem !default;
+$modal-title-line-height:$line-height-base !default; $modal-content-color:null
+!default; $modal-content-bg:$white !default;
+$modal-content-border-color:rgba($black, .2) !default;
+$modal-content-border-width:$border-width !default;
+$modal-content-border-radius:$border-radius-lg !default;
+$modal-content-inner-border-radius:subtract($modal-content-border-radius,
+$modal-content-border-width) !default;
+$modal-content-box-shadow-xs:$box-shadow-sm !default;
+$modal-content-box-shadow-sm-up:$box-shadow !default; $modal-backdrop-bg: $black
+!default; $modal-backdrop-opacity: .5 !default; $modal-header-border-color:
+$border-color !default; $modal-footer-border-color: $modal-header-border-color
+!default; $modal-header-border-width: $modal-content-border-width !default;
+$modal-footer-border-width: $modal-header-border-width !default;
+$modal-header-padding-y: $modal-inner-padding !default; $modal-header-padding-x:
+$modal-inner-padding !default; $modal-header-padding: $modal-header-padding-y
+$modal-header-padding-x !default; // Keep this for backwards compatibility
+$modal-sm: 300px !default; $modal-md: 500px !default; $modal-lg: 800px !default;
+$modal-xl: 1140px !default; $modal-fade-transform: translate(0, -50px) !default;
+$modal-show-transform: none !default; $modal-transition: transform .3s ease-out
+!default; $modal-scale-transform: scale(1.02) !default; {% endcapture %} {%
+capture modal-default %}
+<div class="modal-dialog">
+  <div class="modal-content">
+    <div class="modal-header">
+      <h3 class="modal-title" id="exampleModalLabel">
+        <i class="bi bi-exclamation-circle left me-2"></i>Modal title
```

Comment:
Replace me-2 to me-3
```suggestion
        <i class="bi bi-exclamation-circle left me-3"></i>Modal title
```